cmd/omegaup-grader/frontend_handler.go

```diff
@@ -320,6 +327,36 @@ func broadcastRun(
 	if err != nil {
 		return err
 	}
+	var groupName string
+	var scoreByGroup float64
+	rows, error := queryWithRetry(
```

Comment:
```suggestion
	rows, err := queryWithRetry(
```

el nombre idiomático del error en Go es `err`.

Reply:
Listo